tasks/cert.yaml

```diff
@@ -1,11 +1,9 @@
 ---
-
-- set_fact: _letsencrypt_certbot_args="{{letsencrypt_certbot_args + ['--renew-by-default']}}"
-  when: letsencrypt_force_renew == true
-
-- set_fact: _letsencrypt_certbot_args="{{letsencrypt_certbot_args + ['--keep-until-expiring']}}"
-  when: letsencrypt_force_renew != true
-
+- set_fact: _letsencrypt_certbot_args="{{ letsencrypt_certbot_args + ['--renew-by-default'] }}"
+  when: letsencrypt_force_renew
+- set_fact: _letsencrypt_certbot_args="{{ letsencrypt_certbot_args + ['--keep-until-expiring'] }}"
+  when: not letsencrypt_force_renew
+- set_fact: _letsencrypt_certbot_combined_args="{{_letsencrypt_certbot_args + letsencrypt_certbot_default_args + letsencrypt_certbot_extra_args}}"
```

Comment:
According to my previous comment, please rename `letsencrypt_certbot_extra_args` to `letsencrypt_certbot_args`